### ISSUE TITLE
Any information about the difference between roi pooling, roi crop and roi align?

### ISSUE THREAD

**Issue by askerlee** [jwyang/faster-rcnn.pytorch#206]
Anyone could give some brief explanations or point to other references? I know what roi pooling works, but I'm not sure about roi crop & roi align. Thank you very much.

**Comment by jwyang** [comment]
hi, @askerlee , 

roi align is from mask r-cnn: https://arxiv.org/pdf/1703.06870.pdf

roi crop is the bilinear interpolation as used in spatial transformer network: https://arxiv.org/pdf/1506.02025.pdf

let me know if you have any further problems.

**Comment by askerlee** [comment]
Thank you! Sounds roi crop is cool. Have you tested its performance on the few public datasets?

**Comment by jwyang** [comment]
@askerlee yes, i tested and it indeed performs better on pascal voc and coco.

**Comment by cwlinghk** [comment]
Both roi crop and align are using bilinear interpolation. How are they different from each other?
